RD-6444 Update documentation covering blueprint deployment (#2519)
cloudify-cosmo/docs.getcloudify.org: content/working_with/console/widgets/deployments.md

on the right of every deployment allows performing the following operations:

* Install/Uninstall deployment

> * Deploy blueprint on deployment - this option is only available when the deployment is containing `csys-obj-type: environment` label

Reviewer: In other places you are referring to environment, so maybe here you can do it as well:
```suggestion
* Deploy blueprint on environment - this option is only available when selected deployment is an environment
```
?

BTW I hope in the future https://cloudifysource.atlassian.net/browse/RD-4113 will be implemented, so all mentions of `environment` can be linked to proper page.

Author: I'd be fine with that direction 💪 
But the issue is that the PR has been merged before I could even consider doing it 😅 
Sorry about that
